.gitignore

```diff
@@ -65,5 +67,4 @@ central.tar.gz
 # I'm storing server output here, don't want to push it to git
 example_output
 test_auth
-/bin
```

Comment:
Why is `/bin` being removed? I use it for commands specific to me and others will want to use it for commands for them.

Reply:
It is a duplicate, line 65 has it there too

Reply:
I think I removed the duplicate line in a separate change while you were working on this, I can't see the duplicate in my gitignore.

Reply:
Ok I will undo this change then